src/js/components/picklist/index.js

```diff
@@ -53,11 +53,16 @@ const PicklistEmpty = View.extend({
   className: 'picklist__message',
   template: hbs`
     {{#if isLoading}}
-      {{ loadingText }}
+      <div class="picklist__message-loading js-loading">
+        {{fas "circle"}}{{ loadingText }}
+      </div>
     {{else}}
       {{ noResultsText }}
```

Comment:
what if we wrapped this in a div class="picklist__message"  and removed it from the view className?